### PR TITLE
Update test cases for invalid commands in stages 2, 4, 6, and 7

### PR DESCRIPTION
**Pull request by andy1li**

### CODE REVIEW

File: internal/stage6.go [hunk 1]
```diff
@@ -33,7 +34,7 @@ func testType1(stageHarness *test_case_harness.TestCaseHarness) error {
 		}
 	}
 
-	invalidCommands := []string{"nonexistent", "nonexistentcommand"}
+	invalidCommands := random.RandomWords(2)
```
Comment (rohitpaulk) [inline internal/stage6.go:37]: We can't really "guarantee" that these are invalid btw - for example, "cat" might be a random word that is returned. Not sure if this is too verbose, but how about something like `invalid_command_<random>`?

**Review by rohitpaulk — CHANGES_REQUESTED** [review]
Added a note re: how we ensure a random word is not a valid command

File: internal/stage6.go (hunk 1 above)
Comment (andy1li) [reply internal/stage6.go:37]: @rohitpaulk How about `<random>_command`?

Comment (rohitpaulk) [reply internal/stage6.go]: @andy1li yep, works!

Comment (rohitpaulk) [reply internal/stage6.go]: Just went through the fixtures and what you have right now looks good btw! No need to change further

**Review by rohitpaulk — APPROVED** [review]

**Pull request merged**